src/views/guidelines/guidelines.jsx

```diff
@@ -16,27 +16,28 @@ const Guidelines = () => (
             }
         >
             <p>
-                <FormattedMessage
-                    className="intro"
-                    id="guidelines.header"
-                />
+                <FormattedMessage id="guidelines.header1" />&nbsp;
+                <strong><FormattedMessage id="guidelines.header2" /></strong>&nbsp;
```

Comment:
Having these as separate phrases is actually helpful. When they're all one object and we change a small part the whole thing has to be retranslated. If we change one of the sentences the others stay translated.